streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordCollectorTest.java

```diff
@@ -506,103 +445,12 @@ public void shouldThrowStreamsExceptionOnSubsequentCallIfFatalEvenWithContinueEx
         assertThat(thrown.getMessage(), equalTo("Error encountered sending record to topic topic for task 0_0 due to:\norg.apache.kafka.common.errors.AuthenticationException: KABOOM!\nWritten offsets would not be recorded and no more records would be sent since this is a fatal error."));
     }
 
-    @Test
-    public void shouldThrowTaskMigratedExceptionOnCommitFailed() {
```

Comment:
move all 4 tests to `TaskManagerTest`

Reply:
Checkmark for proving the 6 tests are all migrated.